slices/libx264-163.yaml

```diff
@@ -0,0 +1,8 @@
+package: libx264-163
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+    contents:
+      /usr/lib/*-linux-*/libx264.so.163*:
```

Comment:
We should probably include the following file too, only for `arch: i386`:

```
/usr/lib/i386-linux-gnu/i686/sse2/libx264.so.163
```

From https://packages.ubuntu.com/jammy/i386/libx264-163/filelist.

Reply:
added